test/1_stdlib/TestCharacterSet.swift

```diff
@@ -103,26 +103,26 @@ class TestCharacterSet : TestCharacterSetSuper {
     }
     
     func testInsertAndRemove() {
-        var asciiUppercase = CharacterSet(charactersIn: UnicodeScalar(0x41)...UnicodeScalar(0x5A))
-        expectTrue(asciiUppercase.contains(UnicodeScalar(0x49)))
-        expectTrue(asciiUppercase.contains(UnicodeScalar(0x5A)))
-        expectTrue(asciiUppercase.contains(UnicodeScalar(0x41)))
-        
-        asciiUppercase.remove(UnicodeScalar(0x49))
-        expectTrue(!asciiUppercase.contains(UnicodeScalar(0x49)))
-        expectTrue(asciiUppercase.contains(UnicodeScalar(0x5A)))
-        expectTrue(asciiUppercase.contains(UnicodeScalar(0x41)))
+        var asciiUppercase = CharacterSet(charactersIn: UnicodeScalar(0x41)!...UnicodeScalar(0x5A)!)
+        expectTrue(asciiUppercase.contains(UnicodeScalar(0x49)!))
+        expectTrue(asciiUppercase.contains(UnicodeScalar(0x5A)!))
+        expectTrue(asciiUppercase.contains(UnicodeScalar(0x41)!))
+        
+        asciiUppercase.remove(UnicodeScalar(0x49)!)
+        expectTrue(!asciiUppercase.contains(UnicodeScalar(0x49)!))
+        expectTrue(asciiUppercase.contains(UnicodeScalar(0x5A)!))
+        expectTrue(asciiUppercase.contains(UnicodeScalar(0x41)!))
        
 
         // Zero-length range
-        asciiUppercase.remove(charactersIn: UnicodeScalar(0x41)..<UnicodeScalar(0x41))
-        expectTrue(asciiUppercase.contains(UnicodeScalar(0x41)))
+        asciiUppercase.remove(charactersIn: UnicodeScalar(0x41)!..<UnicodeScalar(0x41)!)
+        expectTrue(asciiUppercase.contains(UnicodeScalar(0x41)!))
 
-        asciiUppercase.remove(charactersIn: UnicodeScalar(0x41)..<UnicodeScalar(0x42))
-        expectTrue(!asciiUppercase.contains(UnicodeScalar(0x41)))
+        asciiUppercase.remove(charactersIn: UnicodeScalar(0x41)!..<UnicodeScalar(0x42)!)
+        expectTrue(!asciiUppercase.contains(UnicodeScalar(0x41)!))
         
         asciiUppercase.remove(charactersIn: "Z")
-        expectTrue(!asciiUppercase.contains(UnicodeScalar(0x5A)))
+        expectTrue(!asciiUppercase.contains(UnicodeScalar(0x5A)!))
```

Comment:
Same here, I would expect `UnicodeScalar(0x5A)` to just call `UnicodeScalar(UInt8)`.